core/src/com/fallenflame/game/LevelController.java

```diff
@@ -56,9 +61,10 @@
 
     // Controllers
     /** Light Controller */
-    private LightController lightController;
+    private final LightController lightController;
```

Comment:
Why this change?

Reply:
@foxhatleo

Reply:
IntelliJ suggested and highlighted it with yellow background so I might have just clicked on the suggestion. I can revert it, altho it doesn't matter cos we don't create controller instance twice.